### DISCUSSION TITLE
is feathers dove stable ?

### DISCUSSION THREAD

**Discussion by noor-tg** [feathersjs/feathers#2439] [Q&A] [answered]
Is feathersjs dove version stable for use in Production ?
if not, what is needed for it to be stable ?

**Comment by daffl** [comment, marked as answer]
Yes, it is already used in production, but the more people that upgrade and report any issues they might have the better. Other than the breaking changes outlined in the [migration guide](https://dove.docs.feathersjs.com/guides/migrating.html) everything should be backwards compatible, however the major _new_ features are still under development and a brand new v5 app will look quite a bit different. You can follow the milestone at https://github.com/feathersjs/feathers/milestone/11

**Reply by noor-tg** [reply]
thank you. feathersjs is very good project.

**Reply by noor-tg** [reply]
what about the plugins . like database adapters .... and so on ?

**Reply by daffl** [reply]
Due to the backwards compatibility all existing database adapters should still work the same but they also need to be upgraded with the new features (especially types).
